src/components/views/articles.vue

```diff
@@ -0,0 +1,31 @@
+<template>
+    <div>
+        <div class="galery">
```

Comment:
Spelling change to "gallery"
```suggestion
        <div class="gallery">
```